.devcontainer/devcontainer.json

```diff
@@ -47,6 +51,35 @@
 			"-I/workspaces/magma/lte/gateway/c/oai/tasks/nas/esm/msg/",
 			"-I/workspaces/magma/lte/gateway/c/oai/tasks/nas/ies/",
 			"-I/workspaces/magma/lte/gateway/c/oai/tasks/nas/util/"
+		],
+		"files.watcherExclude": {
+			"**/.bazel-cache/**": true,
+			"**/.bazel-cache-repo/**": true,
+		},		
+		"bsv.bazel.buildFlags": [
+			"--config=devcontainer",
+		],
+		"bsv.bazel.testFlags": [
+			"--compilation_mode=dbg",
+		],
+		"bsv.bes.enabled": false,
+		"bsv.bzl.codesearch.enabled": false,
```

Comment:
what does codesearch.enabled do? Sounds interesting.

Reply:
It's code search within a Bazel target. So it's like a fancier search functionality within a scope. It might require a subscription, though. Should double check :) I was just turning off all non-core features when testing the settings.